src/lib/temple/front/baking.ts

```diff
@@ -143,15 +145,15 @@ export function useKnownBakers(suspense = true) {
   const net = useNetwork();
   const { data: bakers } = useRetryableSWR(
     net.type === "main" ? "all-bakers" : null,
-    getAllBakers,
+    getAllBakersBakingBad,
     {
       refreshInterval: 120_000,
       dedupingInterval: 60_000,
       suspense,
     }
   );
 
-  return useMemo(() => (bakers && bakers.length > 1 ? bakers : null), [bakers]);
+  return useMemo(() => (bakers && bakers.length > 1 ? bakers as BakingBadBaker[] : null), [bakers]);
```

Comment:
create type guard inside `getAllBakersBakingBad` fn to prevent this assertion

Reply:
updated